packages/wonder-blocks-link/src/components/link-core.tsx

```diff
@@ -68,9 +70,12 @@ export default class LinkCore extends React.Component<Props> {
             "data-test-id": testId,
             style: [defaultStyles, style],
             target,
+            startIcon,
+            endIcon,
```

Comment:
I don't think you need to add `startIcon` and `endIcon` to `commonProps` since they've already been used to render the props here and no longer need to continue being passed into the `StyledLink` and `StyledAnchor` components.